src/components/Menu/MenuItem/MenuItem.module.scss

```diff
@@ -76,11 +76,3 @@
   flex-grow: 1;
   padding-block: 1px;
 }
-
-.label {
```

Comment:
Unused, designers don't want ellipsis on the Label, it should be a single short word for this case, they will add a task to add a use case